docs/design-logging_error_handling.html

```
@@ -186,7 +495,7 @@ <h1>Logging & Error Handling</h1>
 <div class="row">
     <div class="col-md-8">
         
-        <h2 id="setting-up-serilog">Setting up SeriLog <a id="setting-up-serilog" href="#setting-up-serilog" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Setting up SeriLog"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
+        <h2 id="setting-up-serilog" class="content-heading">Setting up SeriLog <a id="setting-up-serilog" href="#setting-up-serilog" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Setting up SeriLog"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
```

HTMLHint (1.9.2)

[error] The id value [ setting-up-serilog ] must be unique.

(id-unique)